.github/workflows/codecov.yaml

```diff
@@ -14,26 +14,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: actions/setup-go@v3
+      - uses: actions/setup-go@v4
```

Comment:
I'm assuming v4 does the caching bit automatically now? Could you provide some doc for this?

Reply:
Will add in the ref. section